.env.dev

```diff
@@ -1,2 +1,3 @@
 VITE_API_URL=http://localhost:8002/api/v1
-VITE_SITE_TITLE=MaveDB (local API)
```

Comment:
this is a little awkward, because currently this is used as a switch to switch "which API backend am I using while developing?" so when I'm developing only the frontend locally, I run `MODE=live npm run dev` and don't have to run a local backend.

I think we do want a mode where these links point to localhost but the prod api is used instead of a local api. I think the best solution is to have 3 modes:
1. local api and app (dev)
2. prod api, local app (prodapi or some better name)
3. prod api, prod app (live)

and update the readme accordingly.

sorry for the hassle!